packages/runtime/src/interpreter/opcode.ts

```diff
@@ -133,6 +135,20 @@ export class Op {
     return b;
   }
 
+  /**
+   * asserts if given variable type is address
+   * @param a - value to assert (should in bytes)
+   * @param line - line number in TEAL file
+   */
+  assertAddress (a: unknown, line: number): Uint8Array {
```

Comment:
```suggestion
  assertAddress (a: Uint8Array, line: number): Uint8Array {
```

Reply:
In my view, `unknow` is better because `a` can be `int` or `bytes`(stack element). By accident, they can write contracts like 
`
int 9 
itxn_field AssetCloseTo
` 
we also can't compile if we Uint8Array.